src/pages/Quiz/Instructions.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import "./quiz.css";
+
+type InstructionsPropsType = {
+  setStart: (value: boolean) => void;
+  setInstructions: (value: boolean) => void;
+};
+
+export function Instructions({ setStart, setInstructions }: InstructionsPropsType) {
+  const playBttnHandler = () => {
```

Comment:
PlayBttnHandler does not make sense, please update the function to a name which describes the function's action

Reply:
updated